patch mkfs.minix.tests to have correct md5sum on big endian platform

Upstream-Status: Accepted, expected in next release

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>

diff --git a/testsuite/mkfs.minix.tests b/testsuite/mkfs.minix.tests
index 8a33c16..7eecaf2 100755
--- a/testsuite/mkfs.minix.tests
+++ b/testsuite/mkfs.minix.tests
@@ -8,6 +8,14 @@
 
 # testing "test name" "options" "expected result" "file input" "stdin"
 
+# '\n' produces 10 on little endian, but not on big endian
+cr=`echo | od -i | sed 's/.* //g;2d'`
+if [ x"$cr" = x"10" ]; then
+	hash=4f35f7afeba07d56055bed1f29ae20b7
+else
+	hash=5adbc1b3ccd20ca5d0ab5bc1e13ac3fc
+fi
+
 testing "mkfs.minix" \
 	"dd if=/dev/zero of=input bs=1k count=1024 2>/dev/null; mkfs.minix input; md5sum <input" \
 "352 inodes\n"\
@@ -15,7 +23,7 @@ testing "mkfs.minix" \
 "Firstdatazone=15 (15)\n"\
 "Zonesize=1024\n"\
 "Maxsize=268966912\n"\
-"4f35f7afeba07d56055bed1f29ae20b7  -\n" \
+"$hash  -\n" \
 	"" \
 	""
 
